src/module/apps/acc_diff/plugins/brutal3.ts

```diff
@@ -0,0 +1,73 @@
+import { AccDiffHudData, AccDiffHudTarget } from "../data";
+import { AccDiffHudCheckboxPluginData, AccDiffHudPluginCodec } from "./plugin";
+import { enclass } from "../../serde";
+import { slugify } from "../../../util/lid";
+import { AbstractTalent } from "./abstractTalents";
+import { getHistory } from "../../../util/misc";
+import { HistoryHitResult } from "../../../combat/lancer-combat-history";
+//Automated
+
+//Is this laggy?
+function getMisses(actorId?: string | null): HistoryHitResult[] {
+  if (actorId === undefined) return [];
+
+  const actions = getHistory()?.getAllActions(actorId) ?? [];
```

Comment:
Might be preferable to use the actor's UUID instead of plain ID. Should be functionally the same, but can give us a bit more to go on for troubleshooting/debugging.